DP02_13a_Image_Cutout_SciDemo.ipynb

```diff
@@ -0,0 +1,1286 @@
+{
```

Comment:
<p>I feel like there are lots of imports that aren't used? Make sure and use "flake8-nb" to check this before merging. (I see butler imported twice, too.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>You are right, I missed that. flake8 run in the notebook did not flag any of these as unused (still not sure why my command line flake8-nb doesn't work...) I think I successfully removed all of the freeloading packages now.</p>